.gitignore

```diff
@@ -22,6 +28,15 @@ slprj/
 # Matlab figures
 *.fig 
 
-# OS X Finder files
-.DS_Store
+## Sublime Text: ------------------------------------
+##
+## Ignore projects and workspaces.
+##---------------------------------------------------
+*.sublime-project
+*.sublime-workspace
+
+## User specific files: -----------------------------
+##
+##---------------------------------------------------
+speed_test.m
```

Comment:
User specific file should not go in the .gitignore, see http://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file for alternative strategies, thanks!